project/MicroServiceBuild.scala

```diff
@@ -36,33 +33,18 @@ private object AppDependencies {
   object Test {
     def apply() = new TestDependencies {
       override lazy val test = Seq(
-        "uk.gov.hmrc" %% "hmrctest" % hmrcTestVersion % scope,
+        "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
         "org.scalatest" %% "scalatest" % "2.2.6" % scope,
         "org.pegdown" % "pegdown" % "1.5.0" % scope,
         "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
-        "org.mockito" % "mockito-all" % mockito,
-        "org.specs2" % "specs2_2.10" % specs2,
-        "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.1"
+        "org.mockito" % "mockito-core" % "3.3.3",
+        "org.specs2" % "specs2_2.10" % "2.3.13",
+        "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.3"
+//        "com.github.tomakehurst"    % "wiremock-jre8"       % "2.21.0"          % scope
```

Comment:
Should this wiremock dependency be commented out? If it doesn't need to be added it can deleted.